frontend/packages/operator-lifecycle-manager/src/components/catalog-source.tsx

```diff
@@ -112,77 +111,145 @@ const DefaultSourceKebab: React.FC<DefaultSourceKebabProps> = ({
         </div>
-        <div className="col-xs-4">
-          <dl className="co-m-pane__details">
-            <dt>{t('catalog-source~Publisher')}</dt>
-            <dd data-test-id="catalog-source-publisher">{obj.spec.publisher}</dd>
-          </dl>
+        <div className="col-sm-6 col-xs-12">
+          <div className="co-m-pane__body">
+            <DetailsItem
+              editAsGroup
+              label="Status"
+              obj={catalogSource}
+              path="status.connectionState.lastObservedState"
+            />
+            <DetailsItem label="Display Name" obj={catalogSource} path="spec.displayName" />
+            <DetailsItem label="Publisher" obj={catalogSource} path="spec.publisher" />
+            <DetailsItem label="Availability" obj={catalogSource} path="metadata.namespace">
+              {catsrcNamespace}
+            </DetailsItem>
+            <DetailsItem label="Endpoint" obj={catalogSource} path="spec.image">
+              {getEndpoint(catalogSource)}
+            </DetailsItem>
+            <DetailsItem
+              label="Registry Poll Interval"
+              obj={catalogSource}
+              path="spec.updateStrategy.registryPoll.interval"
+              canEdit={!_.isEmpty(catalogSource.spec.updateStrategy)}
+              onEdit={() => editRegitryPollInterval({ catalogSource })}
+            />
+            <DetailsItem label="Number of Operators" obj={catalogSource} path="">
+              {operatorCount}
+            </DetailsItem>
+          </div>
         </div>
       </div>
-      <div className="co-m-pane__body">
-        <SectionHeading text={t('catalog-source~Packages')} />
-        <PackageManifestList
-          loaded
-          data={packageManifests}
-          operatorGroup={toData(operatorGroups)}
-          subscription={toData(subscriptions)}
-        />
-      </div>
     </div>
   ) : (
     <div />
   );
 };
 
-export const CatalogSourceDetailsPage: React.FC<CatalogSourceDetailsPageProps> = (props) => (
-  <DetailsPage
-    {...props}
-    namespace={props.match.params.ns}
-    kind={referenceForModel(CatalogSourceModel)}
-    name={props.match.params.name}
-    pages={[navFactory.details(CatalogSourceDetails), navFactory.editYaml()]}
-    menuActions={Kebab.factory.common}
-    resources={[
-      {
-        kind: referenceForModel(PackageManifestModel),
-        isList: true,
-        namespace: props.match.params.ns,
-        selector: { matchLabels: { catalog: props.match.params.name } },
-        prop: 'packageManifests',
-      },
-      {
-        kind: referenceForModel(SubscriptionModel),
-        isList: true,
-        namespace: props.match.params.ns,
-        prop: 'subscriptions',
-      },
-      {
-        kind: referenceForModel(OperatorGroupModel),
-        isList: true,
-        namespace: props.match.params.ns,
-        prop: 'operatorGroups',
-      },
-    ]}
-  />
-);
+export const Operators: React.FC<PackageManifestPageProps> = (props) => {
```

Comment:
This should be a more descriptive name like `CatalogSourceOperatorsPage`. Exporting a module with a generic name like `Operators` could create confusion when importing modules in other areas of the app.

Reply:
makes sense